.mailmap

```diff
@@ -0,0 +1,5 @@
+Lee Clemens <java@leeclemens.net>
+Serg G. Brester <info@sebres.de>
+Serg G. Brester <serg.brester@sebres.de>
+Serg G. Brester <sergey.brester@W7-DEHBG0189.wincor-nixdorf.com>
```

Comment:
that was actually the point of having this since without it you would double as 
```
$> git shortlog -sn | grep -i bre
    75	Serg G. Brester
     1	sebres
```
and since that is how you got into the git history -- it will now be there forever and ever! ;)